lib/ttnt/test_selector.rb

```diff
@@ -41,5 +47,16 @@ def select_tests
       end
       tests.delete(nil)
     end
+
+    private
+
+    # Find the commit `rake ttnt:test:anchor` has been run on.
+    #
+    # @param sha [String] sha of a commit from which search starts
+    def find_anchored_commit(sha)
+      ttnt_tree = @repo.lookup(@repo.lookup(sha).tree['.ttnt'][:oid])
+      anchored_sha = @repo.lookup(ttnt_tree['commit_obj.txt'][:oid]).content
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.